fenicsprecice/fenicsprecice.py

```diff
@@ -139,20 +141,14 @@ def get_point_sources(self, data):
             "PointSources only supported for vector valued read data."
 
         for v in data.keys():
-            assert (len(v) == self._fenics_dimensions), \
+            assert (len(v) == self._interface.get_dimensions()), \
```

Comment:
Also good. Makes code easier to understand and we have to maintain fewer members.